mobile/pubspec.yaml

```diff
@@ -36,7 +36,7 @@ dependencies:
   path_provider: ^2.0.14
   animations: ^2.0.7
   provider: ^6.0.5
-  intl: ^0.17.0
+  intl: ^0.18.0
```

Comment:
@NoahNsimbe I also changed this because the build was failing on my end when i used the initial one